pkgs/data/icons/quintom-cursor-theme/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenvNoCC, fetchFromGitLab, lib }:
+stdenvNoCC.mkDerivation {
+  pname = "quintom-cursor-theme";
+  version = "2019-10-24";
```

Comment:
```suggestion
  version = "unstable-2019-10-24";
```

Reply:
What is the reasoning behind this? The author gives no indication that master is their development branch, and it has been stable for ~2 years.

Reply:
From the [Nixpkgs Manual](https://nixos.org/manual/nixpkgs/stable/#sec-package-naming):
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

Reply:
aha, thanks for that explanation. fixed.